### PR TITLE
chore: remove the usages of raw target closed message constant

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-core/src/common/errors.ts [hunk 1]
```diff
@@ -25,16 +25,13 @@ class CustomError extends Error {
 
 export class TimeoutError extends CustomError {}
 
-export const kTargetClosedErrorMessage = 'Target page, context or browser has been closed';
-export const kTargetCrashedErrorMessage = 'Target crashed';
-
 export class TargetClosedError extends Error {
```
Comment (dgozman) [inline packages/playwright-core/src/common/errors.ts:28]: Should this be `extends CustomError` similarly to `TimeoutError`?

File: packages/playwright-core/src/common/errors.ts [hunk 2]
```diff
@@ -25,16 +25,13 @@ class CustomError extends Error {
...
-    super(kTargetClosedErrorMessage);
+  constructor(cause?: string, logs?: string) {
+    super((cause || 'Target page, context or browser has been closed') + (logs || ''));
     this.name = this.constructor.name;
   }
 }
 
 export function isTargetClosedError(error: Error) {
-  return error instanceof TargetClosedError || error.message.includes(kTargetClosedErrorMessage);
+  return error instanceof TargetClosedError || error.name === 'TargetClosedError';
```
Comment (dgozman) [inline packages/playwright-core/src/common/errors.ts:36]: Given the change in `serializers.ts`, I wonder when does `error.name ===` code path trigger?

File: packages/playwright-core/src/utils/manualPromise.ts [hunk 3]
```diff
@@ -99,8 +101,8 @@ export class LongStandingScope {
     const terminatePromise = new ManualPromise<Error>();
     if (this._terminateError)
       terminatePromise.resolve(this._terminateError);
-    if (this._terminateErrorMessage)
-      terminatePromise.resolve(new Error(this._terminateErrorMessage));
+    if (this._closeError)
+      terminatePromise.resolve(this._closeError);
```
Comment (dgozman) [inline packages/playwright-core/src/utils/manualPromise.ts:105]: Ideally, this would be the same `errorClone` as above, to preserve `frames`.

**Review by dgozman — APPROVED** [review]

File: packages/playwright-core/src/common/errors.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright-core/src/common/errors.ts:28]: I have no idea!

File: packages/playwright-core/src/common/errors.ts (hunk 2 above)
Comment (pavelfeldman) [reply packages/playwright-core/src/common/errors.ts:36]: That is for cloned errors.

**Pull request merged**